addons/website/static/src/interactions/carousel/carousel_slider.js

```diff
@@ -93,7 +93,7 @@ export class CarouselSlider extends Interaction {
     onSlideCarousel(ev) {
         const imageEls = [...this.carouselInnerEl.querySelectorAll("img")];
         const isLoading = imageEls.some(el => el.loading !== "lazy" && !el.complete);
-        if (isLoading) {
+        if (isLoading && this.options.scrollMode === "single") {
```

Comment:
This is not the right fix, because we want to preload the images for all the carousels, and not only those that have the "single scroll" mode (it allows to prevent ugly visual effects when sliding) :slightly_smiling_face: 
The right fix here should be something that makes sure that the `slid` event is triggered, as it is what the mutex is waiting for to be released (which is why everything is blocked)

I asked QSM and normally something is supposed to retrigger the `slide` once the images are loaded, you should look into that instead and see why it doesn't happen when adding a new slide containing an image (maybe because the added image is not properly loaded yet ?) :wink: (it can be done in another PR)